src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -906,6 +907,10 @@ namespace ReportSizingManager {
 									TotCapTempModFac = CurveValue ( DataTotCapCurveIndex, CoilInWetBulb, OutTemp );
 								} else {
 									TotCapTempModFac = 1.0;
+									
+									if( SameString( CompType, "Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl" ) ){
+										EnergyPlus::DXCoils::CalcVRFCoilCapModFac( 0, _, CompName, CoilInTemp, _, _, _, TotCapTempModFac);
```

Comment:
@rraustad @mjwitte

It seems we cannot use the method that Rich suggested, i.e., to calculate **DataTotCapCurveValue** with the following codes in *DXCoils.cc*, and then directly use it to assign **TotCapTempModFac** in *ReportSizingManager.cc*.
<pre>
EnergyPlus::DXCoils::CalcVRFCoilCapModFac( 0, _, CompName, CoilInTemp, _, _, _, DataTotCapCurveValue);
</pre>
The problem is that we need air condition at sizing (e.g., CoilInWetBulb and CoilInTemp), which can only be obtained within *ReportSizingManager.cc*
<pre>
==========This is what I coded in <b>ReportSizingManager.cc</b>==========

	if ( DataTotCapCurveIndex > 0 ) {
		TotCapTempModFac = <b>CurveValue</b> ( DataTotCapCurveIndex, <b>CoilInWetBulb</b> , OutTemp );
	} else {
		TotCapTempModFac = 1.0;
		
		if( SameString( CompType, "Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl" ) ){
			<b>CalcVRFCoilCapModFac</b>( 0, _, CompName, <b>CoilInTemp</b>, _, _, _, TotCapTempModFac);
		}
	}
</pre>
If there is no better solution, I think it is fine to use the current design, right? We have to call a method to calculate **TotCapTempModFac ** anyway: current VRF coil uses *CurveValue* method , while the new VRF coil uses a specifically designed method *CalcVRFCoilCapModFac*.